integrations/GA4/utils.js

```diff
@@ -241,11 +240,9 @@ const proceedCloudMode = (rudderElement, measurementId) => {
   let sessionId;
   if (cookieObj[`_ga_${measurementIdArr[1]}`]) {
     sessionId = cookieObj[`_ga_${measurementIdArr[1]}`].split(".");
-    const GA4 = { sessionId: sessionId[2] };
-    payload.message.integrations = { All: true, GA4 };
-    return payload;
+    return { sessionId: sessionId[2] };
```

Comment:
This method should return the session ID value, not the wrapping object. The packaging should be done at the level above.

Reply:
updated